### PR TITLE
[bridge cli] print data in json and print more data

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

1. now you can use `--network testnet` to avoid having to provide proxy address
2. add more stuff to print functions, including nonces
3. print everything in json, for easier piped processing

See offline nodes:
```
sui-bridge-cli view-sui-bridge  --sui-rpc-url https://rpc.testnet.sui.io:443 --ping --hex | jq '.result.committee[] | select(.status == "offline")'
```

See eth nonce:
```
sui-bridge-cli view-eth-bridge --network testnet --eth-rpc-url https://ethereum-sepolia-rpc.publicnode.com
```

See sui nonce:
```
sui-bridge-cli view-sui-bridge  --sui-rpc-url https://rpc.testnet.sui.io:443 --ping --hex | jq '.result.nonces'
```

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: crates/sui-bridge-cli/src/lib.rs [hunk 1]
```diff
@@ -42,13 +42,20 @@ use sui_types::transaction::{ObjectArg, Transaction, TransactionData};
 use sui_types::{TypeTag, BRIDGE_PACKAGE_ID};
 use tracing::info;
 
+pub const SEPOLIA_BRIDGE_PROXY_ADDR: &str = "0xAE68F87938439afEEDd6552B0E83D2CbC2473623";
```
Comment (dariorussi) [inline crates/sui-bridge-cli/src/lib.rs:45]: is this really a constant? 
is it something we need to change for mainnet?
not super clear what this constant is doing here

File: crates/sui-bridge-cli/src/main.rs [hunk 2]
```diff
@@ -431,3 +512,73 @@ async fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[derive(serde::Serialize, Default)]
+struct PrintEthBridge {
```
Comment (dariorussi) [inline crates/sui-bridge-cli/src/main.rs:517]: this is a bit of a strange name to me, but if that is ok with you it's all good. Some for the other "print"...
`EthBridgeSummary`? (not sure that is a good name but felt I had to try something else - lol)

**Review by dariorussi — APPROVED** [review]
thanks

File: crates/sui-bridge-cli/src/lib.rs (hunk 1 above)
Comment (longbowlu) [reply crates/sui-bridge-cli/src/lib.rs:45]: it's constant for each network so for mainnet there will be a new one.

File: crates/sui-bridge-cli/src/main.rs (hunk 2 above)
Comment (longbowlu) [reply crates/sui-bridge-cli/src/main.rs:517]: will update

Comment (longbowlu) [reply crates/sui-bridge-cli/src/main.rs:517]: renamed to `Output`

**Pull request merged**